test/sharness/t0040-add-and-cat.sh

```diff
@@ -223,8 +223,8 @@ test_expect_success "ipfs cat output looks good" '
 	test_cmp expected actual
 '
 
-test_expect_success "ipfs cat accept hash from stdin" '
-	echo "$HASH" | ipfs cat >actual
+test_expect_success "ipfs cat accept hash from built input" '
+	echo "$HASH" | xargs ipfs cat >actual
```

Comment:
I thought this had been reverted, @whyrusleeping @Kubuxu -- the tests on master show it's still here.